app/uk/gov/hmrc/eusubsidycompliancefrontend/views/FinancialDashboardPage.scala.html

```diff
@@ -0,0 +1,119 @@
+@*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.eusubsidycompliancefrontend.config.AppConfig
+@import uk.gov.hmrc.eusubsidycompliancefrontend.controllers.routes
+@import uk.gov.hmrc.eusubsidycompliancefrontend.views.formatters.BigDecimalSyntax._
+@import uk.gov.hmrc.eusubsidycompliancefrontend.views.models.FinancialDashboardSummary
+
+@this(
+  layout: FullWidthLayout,
+  govukTable: GovukTable
+)
+
+@(summary: FinancialDashboardSummary)(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
+
+@layout(
+    pageTitle = Some(messages("dashboard.title")),
+    backLinkEnabled = true,
+    backLink = Some(routes.AccountController.getAccountPage().url)
+) {
+
+    <h1 class="govuk-heading-l">@messages("dashboard.title")</h1>
+
+    <table class="govuk-table">
```

Comment:
i think we have govukSummaryList in our hmrc lib for creating table like this. I may be wrong please do confirm from simon

Reply:
Ok, he's got a subtask to review and tidy up the template so it can be addressed then. Initially I've just taken the markup from the prototype and got it working.

Reply:
👍